option.go

```diff
@@ -355,6 +357,21 @@ func setupBuiltinFunctions(opts ...Option) []Option {
 		Func("string", func(v interface{}) string { return cast.ToString(v) }),
 		Func("int", func(v interface{}) int { return cast.ToInt(v) }),
 		Func("bool", func(v interface{}) bool { return cast.ToBool(v) }),
+		Func("compare", func(x, y interface{}, ignoreKeys ...string) bool {
```

Comment:
Is this the correct interface for the compare function?

Reply:
Looks good to me!